notebooks/dynamicrafter-animating-images/dynamicrafter-animating-images.ipynb

```diff
@@ -0,0 +1,696 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #24.        model = model.cpu()
</pre><p>is there way to specify cpu device during model loading to prevent offloading on cuda? (it can lead to some troubles for running if torch installed without gpu support or user does not have  gpu device)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/2003/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>It was redundant, removed</p>